avarage: true   #the Average of the Last Minute. New info every 5s (Calculated by Kernel)
percent: true   #The Average over frequency/1s in percent
percentPerCore: true  #like percent but for every core
temperature: true   #temperature of the CPU

monitoring:
  frequency: 0.1   #Publish and calculation frequency

monitor_mode : 1
#AggregationStrategies mode: 1 LAST, 2 FIRST, 3 MIN, 4 MAX, 5 AVG#
